stumpui/windows.lisp

```diff
@@ -0,0 +1,215 @@
+;;;; Copyright (C) 2016 Joram Schrijver
```

Comment:
This is `/stumpui/windows.lisp`, not `/window.lisp`. I agree we need a better name, but `window-protocol.lisp` isn't it. This file contains implementations of the protocol, not the actual protocol.